cmd/earth/main.go

```diff
@@ -39,20 +42,23 @@ type earthApp struct {
 }
 
 type cliFlags struct {
-	buildArgs         cli.StringSlice
-	secrets           cli.StringSlice
-	artifactMode      bool
-	imageMode         bool
-	push              bool
-	noOutput          bool
-	noCache           bool
-	pruneAll          bool
-	pruneReset        bool
-	buildkitdSettings buildkitd.Settings
-	allowPrivileged   bool
-	buildkitHost      string
-	buildkitdImage    string
-	remoteCache       string
+	buildArgs           cli.StringSlice
+	secrets             cli.StringSlice
+	artifactMode        bool
+	imageMode           bool
+	push                bool
+	noOutput            bool
+	noCache             bool
+	pruneAll            bool
+	pruneReset          bool
+	buildkitdSettings   buildkitd.Settings
+	allowPrivileged     bool
+	buildkitHost        string
+	buildkitdImage      string
+	remoteCache         string
+	configPath          string
+	GitUsernameOverride string
+	GitPasswordOverride string
```

Comment:
I believe these can be private, right?

Reply:
definitely, not sure why I made them private. fixed.